src/pages/OrganizationProjectsPage/OrganizationProjectsPage.tsx

```diff
@@ -323,6 +341,7 @@ const OrganizationProjectsPage: React.FC<{}> = ({}) => {
           onCreateClick={() => updateCreateModelVisibility(true)}
           permissions={['projects/create']}
           path={[`/${orgLabel}`]}
+          supTitle="Organization"
```

Comment:
Can we rename `supTitle`. I may be wrong but it looks like the value of this is only ever `Organization`. If that's the case then maybe `Organization` or `OrgTitle` may be more descriptive (we could also make this a boolean instead and if it is true display the string "Organization" in the component). 

If there are other possible values for this, then I would suggest the name `RouteTitle`, `RouteLabel`, `AccessTitle` or something along those lines. The `sup` in `supTItle` is confusing for me.

Reply:
the RouteHeader components is global component so it's preferably to not make special cases in it, 
and the idea behind the naming is like `<sup>` html tag, so it will be only on top of the title,
and since the component is `RouteHeader`, i think properties must not include the prefix of `Route`
since it's self explanatory